.npmignore

```diff
@@ -6,7 +6,6 @@ demos1
 TODO.md
 test
 testlib
-src
```

Comment:
Ah ... let me bring it back. I removed it when I was testing the component locally.